crawl-ref/source/dat/database/miscname.txt

```diff
@@ -136,6 +136,14 @@ The Orb of Zot is coveted for many reasons. Will you seize it for your own?
 A smirking vizier told you that the Orb of Zot is just within this dungeon.
 
 If you return without finding the Orb of Zot, so far below, it's all over!
+
+One, two, three... Ready or not, find the Orb of Zot!
+
+Far below lies the Orb of Zot. Hopefully nothing unpleasant happens on the way there.
+
+It's time to claim the Orb of Zot, or maybe just get slain and rot!
+
+The @_great_adj_@ Orb of Zot should be around here somewhere.
```

Comment:
This is just wrong, unfortunately. Maybe that's the joke, but it's not 'around here somewhere' - it's quite a long way below. Check out the other messages - it's "in the heart of the dungeon", "in the unfathomable depths", "deep within this dungeon". The only one which is a little more misleading is "A smirking vizier told you that the Orb of Zot is just within this dungeon", and that one pretty explicitly presents itself as being from an untrustworthy source.

As the first message we give a new player, let's not lie to them without any warning.